Source/View Controller/ProfileViewController/ProfileImageViewControllerExtension.swift

```diff
@@ -33,16 +33,17 @@ extension ProfileViewController: UIImagePickerControllerDelegate, UINavigationCo
         let imagePickerViewController = UIImagePickerController()
         imagePickerViewController.delegate = self
         imagePickerViewController.allowsEditing = true
-        imagePickerViewController.sourceType = .photoLibrary
+        imagePickerViewController.sourceType = source
         self.present(imagePickerViewController, animated: true, completion: nil)
     }
     
     // MARK: - Camera Picker Delegate functions
     
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
         if let imageSelected = info[UIImagePickerController.InfoKey.originalImage] as? UIImage {
```

Comment:
This should be `UIImagePickerController.InfoKey.editedImage` instead of `UIImagePickerController.InfoKey.originalImage`. The editedImage part allows us to only take the image that the user puts in the square box.

Reply:
😂yeah I had to actually delete some images before doing that... Minor changes have been made.